src/package-hoister.js

```diff
@@ -291,8 +326,24 @@ export default class PackageHoister {
       }
     }
 
+    const peerDependencies = Object.keys(info.pkg.peerDependencies || {});
```

Comment:
I feel like the indirection would make the code less clear. Why make a function call to an helper when you can just call the function this helper calls? It only matters if we plan to change the `info.pkg` underlying type and suddenly it wasn't a mapping of the package.json anymore, but I don't see what we would have to gain in doing that.